manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json

```diff
@@ -777,5 +777,5 @@
   "timezone": "",
   "title": "Incoming Request",
   "uid": "3jeba3R7z",
-  "version": 16
+  "version": 4
```

Comment:
not sure about downgrade the version 🤔

Reply:
Sometimes it happens and I'm sure it doesn't matter

Reply:
Okay 🙌

Reply:
I mean the version seems to be reset when reload by ConfigMap, that's why it happens